gulpfile.js

```diff
@@ -263,22 +252,44 @@ gulp.task('build-ts',
 gulp.task('build-externs',
     () => gulp.src(EXTERNS_FILES).pipe(gulp.dest(`${DEST_DIR}/externs/`)));
 
-// Builds the final JS file for all supported languages.
-gulp.task('build-all-js', buildJsTasks, () => makeDefaultFile('firebaseui'));
 
-// Builds the final JS file for the default language.
-gulp.task('build-js', ['build-js-' + DEFAULT_LOCALE],
-    () => makeDefaultFile('firebaseui'));
+// Builds the core FirebaseUI JS. Generates the gulp tasks
+// build-firebaseui-js-de, build-firebaseui-js-fr, etc.
+repeatTaskForAllLocales(
+  'build-firebaseui-js-$',
+  ['build-externs', 'build-ts', 'build-soy'],
+  buildFirebaseUiJs,
+);
```

Comment:
This looks unchanged. Is there a reason why you moved it? If not, let's revert this and keep the changes needed.